jekyll/_cci2/deploy-android-applications.adoc

```diff
@@ -0,0 +1,333 @@
+---
+categories: [platforms]
+version:
+- Cloud
+- Server v3.x
+---
+= Deploy Android Applications
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn to deploy Android apps through CircleCI. 
+:icons: font
+:toc: macro
+:toc-title:
+
+[#overview]
+== Overview
+
+In this tutorial, you will learn how to automatically deploy your Android app through CircleCI with link:https://fastlane.tools/[Fastlane]. 
+
+To complete this tutorial, you will need the following:
+
+- An Android project, built using link:https://gradle.org/[Gradle]. 
+- Your Android project set up in CircleCI. If this is your first time setting up a project in CircleCI, refer to our <<getting-started#,Quickstart Guide>> for a walkthrough.
+- Fastlane installed and granted access to Google Play Store using credentials from your Google Developers service account. You may follow the instructions to install, set up, and configure fastlane supply in the link:https://docs.fastlane.tools/getting-started/android/setup/[Fastlane Android setup documentation].
+
+NOTE: In the Fastlane setup, you will download a credentials file from your Google Developers service account. This tutorial assumes that you have saved this file in the `fastlane` folder as `api.json`.
```

Comment:
Hiya 👋🏽 and thanks for the guide!

Can I ask why the credentials file would be committed to the repo, which takes pains to make sure signing keys are not committed?

Reply:
I guess the tutorial means that the file was saved but not added to the repo, given that the contents are saved to an envvar later on.  The phrasing could be clarified IMO.

Anyway the real problem at the moment is https://github.com/CircleCI-Public/android-orb/issues/61